src/test/scala/com/microsoft/hyperspace/index/IndexManagerTests.scala

```diff
@@ -80,6 +80,7 @@ class IndexManagerTests extends HyperspaceSuite with SQLHelper {
             s"$systemPath/${indexConfig1.indexName}" +
               s"/${IndexConstants.INDEX_VERSION_DIRECTORY_PREFIX}=0",
             Constants.States.ACTIVE)
+          val xx = actual.equals(expected)
```

Comment:
remove?

Reply:
Thanks, removed it - I had missed it during debug clean-up.